Common/Extensions/Bundle.swift

```diff
@@ -15,4 +15,7 @@ public extension Bundle {
         return self.infoDictionary?["URL_SCHEME"] as? String
     }
 
+    var courseOpen: String? {
```

Comment:
I would like to see better naming for this variable so that the usage as an activity type is reflected. Maybe `ActityTypeCourseOpen` or `courseOpneActivityType`?